src/list/whole/view.rs

```diff
@@ -538,10 +535,43 @@ where
                         ""
                     };
                     let style = Self::style_width_height(ctx, widths, i, varied_width);
+                    let txt = ctx.props().txt.txt.clone();
+                    let onclick_close = {
+                        ctx.link().callback(move |_| Message::CloseModal)
+                    };
 
                     html! {
                         <td class={classes!("list-whole-list-flat", if border { class_border } else { "" })} style={style}>
-                            { Self::view_column(ctx, index, col) }
+                            { self.view_column(ctx, index, col) }
+                            {
+                                if let Some(index) = self.modal_index {
+                                    if let Some(modal_content) = self.modal_content.get(index) {
+                                        let modal_content = Some(Html::from_html_unchecked(AttrValue::from_str(modal_content).expect("AttrValue never returns Err.")));
+                                        let modal_title = if self.title_id == "config" {
+                                            text!(txt, ctx.props().language, "Node - Config")
+                                        } else {
+                                            text!(txt, ctx.props().language, "Node - Draft")
+                                        };
```

Comment:
`Node - Config` and `Node - Draft` should be directly given by users, instead of any intermediary values like `config`.

Reply:
Done